inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/AuditEntityMapper.java

```diff
@@ -59,4 +59,10 @@ List<Map<String, Object>> sumByLogTsAndIp(@Param(value = "ip") String ip,
             @Param(value = "sDate") String sDate,
             @Param(value = "eDate") String eDate,
             @Param(value = "format") String format);
+
+    List<Map<String, Object>> sumGroupByIp(@Param(value = "groupId") String groupId,
+            @Param(value = "streamId") String streamId,
+            @Param(value = "auditId") String auditId,
+            @Param(value = "sDate") String sDate,
+            @Param(value = "eDate") String eDate);
```

Comment:
what's the meaning of s/eDate, it's better to give a clearer name.

Reply:
Fix.